person.rb

```diff
@@ -1,18 +1,31 @@
-class Person
-  attr_reader :id
+require './Interface/nameable'
+require './rental'
+
+class Person < Nameable
+  attr_reader :id, :rentals
```

Comment:
- [ ] Please add the `rentals` variable as an `attr_accessor` it will enable setting and getting it easy.